package.json

```diff
@@ -42,5 +42,8 @@
     "webpack-cli": "^3.3.0",
     "webpack-config-utils": "^2.3.1",
     "webpack-dev-server": "^3.2.1"
+  },
+  "dependencies": {
+    "lodash": "^4.17.15"
```

Comment:
I don't see where this dependency is being used?

Reply:
weird. I was seeing an error in Github for a vulnerable dependency. https://github.com/Paperspace/client-sdk/network/alerts

Reply:
maybe  a child dependency?

Reply:
babel maybe: https://github.com/Paperspace/client-sdk/blob/master/yarn.lock#L25

Reply:
Ah, ok. Then we should upgrade babel core (along with the rest of the babel suite - they follow dependency numbering)